source/2020-01-27-the-ember-times-issue-132.md

```diff
@@ -56,13 +55,33 @@ If you have a great idea for a meetup, you can [sign up here](https://emberconf.
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Try your first Octane & JAM sandwich at EmberConf 🏎🥪🍓](https://emberconf.com/#/speakers/chris-manson)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Surely, you remember him as [the Emberista who built a blog site with Ember & NodeJs on stage in less than 15 mins](https://www.youtube.com/watch?v=GHbjdesHhPA), you heard him talk about [his work on an Ember-powered blog engine](https://www.youtube.com/watch?v=OnDUp2AhuNo) or you
+recall his [jammiest introduction to the JAM (JavaScript, APIs and Markup) stack for humans](https://www.youtube.com/watch?v=eQULGb10MbQ) with [46 mentions of the word "JAM" in total](https://twitter.com/real_ate/status/1207683401446313984)!
+This year, [Chris Manson (@mansona)](https://github.com/mansona) has even more insights into the modern JAM stack to share with you!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+<!--alex disable king-queen-->
+At [this year's EmberConf](https://emberconf.com/) he's going to teach us about [**An Octane-Powered JAM Stack**](https://emberconf.com/#/schedule/day-1_an-octane-powered-jam-stack); how and why the JAM Stack has become so popular and how Ember Octane blends in well with this approach of web development. But that's not all: you can learn hands-on, how to build you first JAM stack site based on Octane and Empress in his EmberConf workshop [**Build and Publish Your Own empress-blog Template**](https://emberconf.com/#/schedule/pre-conf_build-and-publish-your-own-%60empress-blog%60-template).
+
+
+JAM Stack has become a popular phrase in our industry over the last few years with projects like Gatsby and Vuepress gaining popularity at fantastic speeds. As it is a very new tech stack there is still some innovation that can be added to the mix, and Ember Octane has the power to unlock a whole new generation of JAM Stack developers with the Empress project.
```

Comment:
I thought "popular idea" might have a more positive meaning than "popular phrase". The latter reminded me of something hyped unnecessarily.
```suggestion
JAM Stack has become a popular idea in our industry over the last few years. Projects like Gatsby and Vuepress are gaining popularity at fantastic speeds. As it is a new tech stack, there is still some innovation that can be added to the mix, and Ember Octane has the power to unlock a whole new generation of JAM Stack developers with the Empress project.
```

Reply:
Woops, I copy-pasta'd the abstract from the schedule here for reference while writing the section and forgot to delete it again - will edit and thanks for the heads up!